app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<section class="container flex food-placeholder">
+<div class="food flex">
+  <%= link_to new_recipe_path do %>
+    <button>Add Recipe</button>
+  <% end %>
+
+  <% @recipes.each do |recipe| %>
+  <div class='recipe flex'>
+    <div class="recipe-details flex">
+      <h2><%= link_to recipe.name, recipe_path(recipe) %></h2>
```

Comment:
Good job for the design! `Recipes` index page allows the user to delete a recipe if user is not an owner. The requirement here is that only owner of a recipe can delete it.